src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -312,6 +331,16 @@ protected override Encoding StandardOutputEncoding
 
         protected override int ExecuteTool(string pathToTool, string responseFileCommands, string commandLineCommands)
```

Comment:
@cston, does this properly work with those VB PDB issues we ran into not long ago?